ee/debug/shipper/shipper.go

```diff
@@ -107,11 +109,17 @@ func (s *shipper) Write(p []byte) (n int, err error) {
 	// OTOH, if we started request in New() we would know sooner if we had a bad upload url ... :shrug:
 	s.uploadRequestStarted = true
 	s.uploadRequestWg.Add(1)
-	go func() {
+	gowrapper.Go(context.TODO(), s.knapsack.Slogger(), func() {
 		defer s.uploadRequestWg.Done()
 		// will close the body in the close function
 		s.uploadResponse, s.uploadRequestErr = http.DefaultClient.Do(s.uploadRequest) //nolint:bodyclose
-	}()
+	}, func(r any) {
+		s.knapsack.Slogger().Log(context.TODO(), slog.LevelError,
+			"exiting after upload request panic",
```

Comment:
I don't think this log message is accurate -- we aren't exiting. I don't think you need the log message here per my earlier comment about gowrapper handling logging, but if you want to keep it, maybe `upload request panicked` would be a more accurate message?